packages/shared/src/ui/enums/routes.ts

```diff
@@ -4,6 +4,7 @@ export enum Routes {
   Unauthorized = "/unautorized",
   SelfRegister = "/self_register",
   CustomerArea = "/customer_area",
+  ErrorBoundary = "/error_boundary",
```

Comment:
I'd love this URL to be super-obvious. The current one is very obvious if you look at it in the context of this PR, but https://igoriceteam.web.app/error_boundary wouldn't suggest to me what it does.

I propose `/do_trigger_an_error` for this view path.